wgpu-types/src/lib.rs

```diff
@@ -4362,14 +4393,15 @@ impl<L> TextureDescriptor<L> {
     ///
     /// ```rust
     /// # use wgpu_types as wgpu;
-    /// let desc = wgpu::TextureDescriptor {
+    /// let desc: wgpu::TextureDescriptor<(), Vec<wgpu::TextureFormat>> = wgpu::TextureDescriptor {
```

Comment:
```suggestion
    /// # type TextureDescriptor = wgpu::TextureDescriptor<(), &[wgpu::TextureFormat]>
    /// let desc  = TextureDescriptor {
```

To try to make the code look as similar as possible to actual user code